common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3207,7 +3207,7 @@
   },
   "INVALID_URL" : {
     "message" : [
-      "The url is invalid: <url>. If necessary set <ansiConfig> to \"false\" to bypass this error."
+      "The url is invalid: <url>. Use try_parse_url to tolerate invalid URL and return NULL instead."
```

Comment:
Please add quotes to the function name, see some other errors. This is just to keep it in the same state.